app/forms/candidate_interface/degree_year_form.rb

```diff
@@ -27,6 +28,10 @@ def start_year_is_before_the_award_year
       errors.add(:start_year, :greater_than_award_year, date: award_year) if award_year.present? && award_year.to_i < start_year.to_i
     end
 
+    def award_year_is_in_the_future_for_incomplete_degree
+      errors.add(:award_year, :in_the_future) if award_year.present? && degree.predicted_grade? && award_year.to_i < Time.zone.now.year.to_i
```

Comment:
Does this need to be 

```
award_year.to_i <= Time.zone.now.year.to_i
```

It could be Jan 2022 and they graduate in July 2022